include/gx_layers.h

```diff
@@ -48,10 +48,10 @@ typedef struct GraphicsBanks {
 
 void GX_SetBanks(const GraphicsBanks *banks);
 void GX_DisableEngineALayers(void);
-void GX_EngineAToggleLayers(u32 layer_mask, GXLayerToggle enable);
+void GX_EngineAToggleLayers(GXPlaneMask planeMask, GXLayerToggle enable);
```

Comment:
Should we change the names of these functions now that we know theyre plane mask toggles

Reply:
I already made an issue of this, yes we really should